go.work

```diff
@@ -0,0 +1,7 @@
+go 1.23.4
+
+use (
+	.
+	arrow-mapping
```

Comment:
Please rename the directory to `arrow_mapping`

Reply:
I went with `arrowmapping` to stay consistent with the module name (package import name). Does that also work?
See https://github.com/marcboeker/go-duckdb/pull/378

Reply:
I think that arrowmapping matches the go idioms best as well (https://go.dev/blog/package-names)